Tests/Form/DataTransformer/MoneyToLocalizedStringTransformerTest.php

```diff
@@ -52,6 +52,16 @@ public function testDataTransform($locale, $currency, $scale, $grouping, $input,
         $this->assertEquals($output, $transformer->transform($input));
     }
 
+    public function testDataTransformEmptyInput()
```

Comment:
There actually wasn't a test case covering giving `MoneyToLocalizedStringTransformer::transform()` a null value, so this added case also ensures that an empty string is returned in that case.